### ISSUE TITLE
Identifiable conformation triggers identifier_name rule

### ISSUE THREAD

**Issue by zntfdr** [realm/SwiftLint#2880]
As of Swift 5.1, a new [`Identifiable` protocol](https://github.com/apple/swift/blob/62c1cf31d21d32f6db368fff1c1d346b5e616d90/stdlib/public/core/Identifiable.swift#L15) has been added.

To conform to this protocol, [a variable with name `id`](https://github.com/apple/swift/blob/62c1cf31d21d32f6db368fff1c1d346b5e616d90/stdlib/public/core/Identifiable.swift#L21) has to be declared on the conforming type. 

This declaration triggers the `Identifier Name Violation` rule (Variable name should be between 3 and 40 characters long).

Triggering Example
```
$ echo "struct Example: Identifiable { var id: Int { 1 } }" | swiftlint --use-stdin
```

As much as I agree with the rule, since I don't believe there's any other way around it, I suggest to add an exception for this conformation.

**Comment by marcelofabri** [comment]
You can use the configuration file to exclude it:

```yaml
identifier_name:
  excluded: # excluded via string array
    - id
    - URL
```

**Comment by zntfdr** [comment]
Whoa! I didn't know that, many thanks @marcelofabri 😃

**Comment by shkipan** [comment]
Could you give me a hint? I'm still receiving some warnings about identifier_name violation after setting up configuration file

**Comment by rhwood** [comment]
Is there a way to allow the var `id` only if the containing class, struct, or extension conforms to `Identifiable`?
